container/docker/factory.go

```diff
@@ -142,6 +142,10 @@ func ContainerNameToDockerId(name string) string {
 }
 
 func isContainerName(name string) bool {
+	// always ignore .mount cgroup even if associated with docker and delegate to systemd
```

Comment:
cadvisor will log `Error trying to work out if we can handle containerName: error`, which seems misleading. How about changing this so that `CanHandleAndAccept` return `(true, false, nil)` or (false, false, nil)` instead?

I also think the caller of `CanHandleAndAccept` logs way too many errors periodically...but that's another issue.